docs/project-imports/dcr.md

```diff
@@ -0,0 +1,63 @@
+# Moving DCR into the CSNX monorepo
+
+## Context
+
+We want to relocate DCR in the CSNX monorepo.
+
+## What do we hope to achieve?
+
+- make it easier for Guardian product teams to ship by reducing the friction of interdependency between DCR and other Guardian projects (Source, Libs, consent, commercial code etc) 
+- reduce the time spent by dotcom on managing their infrastructure
+- standardise process: _best practice by default_
+
+# Pre-relocation
+
+## Rationalising NPM dependencies across the existing DCR / AR monorepo
+
+The `dotcom-rendering` repo currently contains one yarn workspace (containing DCR and common rendering) and a third, unattached package (AR), whose dependencies are handled using NPM. 
+
+AR consumes some undeclared dependencies from the root yarn workspace.
+
+Common-rendering is only consumed by AR.
+
+We will move all projects into one `yarn` workspace, and resolve any issues around shared dependencies _before_ any migration takes place.
```

Comment:
So to confirm, AR will move from NPM into yarn worskpaces as a workspace/package? or they will all be in one shared worskspace/package?

Reply:
DCR / AR / CR all as packages in one shared Yarn workspace